lib/pages/app_grid/app_grid_dashboard_view.dart

```diff
@@ -19,71 +21,99 @@ class AppGridDashboardView extends StatelessWidget {
-              cursor: SystemMouseCursors.click,
-              onEnter: (_) => controller.hoverColor.value =
-                  AppGridDashboardViewStyle.hoverColor,
-              onExit: (_) {
-                if (!isOpenAppGridDashboard) {
-                  controller.hoverColor.value = Colors.transparent;
-                }
+      child: Row(
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          if (AppConfig.supportUrl.isNotEmpty)
+            TapRegion(
+              behavior: HitTestBehavior.opaque,
+              onTapInside: (_) {
+                UrlLauncher(
+                  context,
+                  url: AppConfig.supportUrl,
+                ).openUrlInAppBrowser();
               },
-              child: ValueListenableBuilder(
-                valueListenable: controller.hoverColor,
-                builder: (context, color, child) {
-                  return PortalTarget(
-                    portalFollower: const SizedBox(
-                      width: AppGridDashboardViewStyle.sizIcAppGrid,
-                    ),
-                    visible: isOpenAppGridDashboard,
-                    child: PortalTarget(
-                      anchor: const Aligned(
-                        follower: Alignment.topRight,
-                        target: Alignment.bottomRight,
-                      ),
-                      portalFollower: ValueListenableBuilder(
-                        valueListenable: controller.linagoraApplications,
-                        builder: (context, linagoraApplications, child) {
-                          if (linagoraApplications == null) return child!;
-                          return AppGridDashboardOverlay(
-                            linagoraApplications,
-                          );
-                        },
-                        child: const SizedBox.shrink(),
-                      ),
-                      visible: isOpenAppGridDashboard,
-                      child: Container(
-                        decoration: BoxDecoration(
-                          color: color,
-                          borderRadius:
-                              AppGridDashboardViewStyle.appGridIconBorderRadius,
-                        ),
-                        padding: AppGridDashboardViewStyle.appGridIconPadding,
-                        child: SvgPicture.asset(
-                          ImagePaths.icApplicationGrid,
-                          width: AppGridDashboardViewStyle.sizIcAppGrid,
-                          height: AppGridDashboardViewStyle.sizIcAppGrid,
-                        ),
-                      ),
-                    ),
-                  );
-                },
+              child: MouseRegion(
+                cursor: SystemMouseCursors.click,
+                child: SizedBox(
+                  width: AppGridDashboardViewStyle.sizeSupportButton,
+                  height: AppGridDashboardViewStyle.sizeSupportButton,
+                  child: SvgPicture.asset(
```

Comment:
Please add tooltip + hover for this icon.

Reply:
Updated 

https://github.com/user-attachments/assets/92df24a7-577e-4a58-9834-3f616ca6f77e